styles/index.less

```diff
@@ -25,16 +25,15 @@ a {
   align-items: center;
 }
 
-#home {
-  margin-top: 60px;
-}
-
 .headline {
-  margin-top: 50px;
-
-  color: grey;
-  font-size: 14px;
-  letter-spacing: 0.7em;
+  margin-top: 55px;
+  margin-bottom: 8px;
+  padding-bottom: 3px;
+  border-bottom: 1px solid #aaa;
+
+  color: #333;
+  font-size: 16px;
```

Comment:
a lot of custom pixel values.

You should consider defining a few standards and using that all over

  @big-text: 42px;
  @small-emphasis: 16px;

otherwise there's no value to using `less`